bootstrap/README.md

```diff
@@ -1,4 +1,4 @@
-# Computing the Bitcoin State
+# Uploading the Bitcoin State
```

Comment:
```suggestion
# Bootstrapping the Bitcoin State
```
I think bootstrapping is the more accurate term, as bootstrapping = computing + uploading